samples/fluent.js

```diff
@@ -42,15 +42,15 @@ var report = function (err, req) {
         referrer: req.header('Referer'),
         userAgent: req.header('User-Agent'),
         remoteIp: req.ip,
-        responseStatusCode: 500
-      }
-    }
+        responseStatusCode: 500,
+      },
+    },
   };
   structuredLogger.emit('errors', payload);
 };
 
 // Handle errors (the following uses the Express framework)
-app.use(function (err, req, res, next) {
+app.use(function(err, req, res) {
```

Comment:
Turns out the `next` argument is required, it's how Express knows this middleware function is an error-handler middleware. Need to add `next` back and disable the linter.

Reply:
Done.